images/daos-server-image.pkr.hcl

```diff
@@ -40,6 +40,11 @@ source "googlecompute" "daos-server-centos-7" {
 build {
   sources = ["source.googlecompute.daos-server-centos-7"]
 
+  provisioner "file" {
```

Comment:
I am wondering if you should copy this file during the image making, or as part of the terraform. Not sure about /tmp, but probably ok.